package.json

```diff
@@ -21,8 +21,10 @@
     "deploy-prod": "serverless --stage=prod deploy",
     "format": "prettier --write es5 './**/*.js'",
     "prepare-production": "snyk protect",
-    "start": "serverless offline start --skipCacheInvalidation",
-    "test": "snyk test"
+    "start": "DEBUG=fcc:* serverless offline start --skipCacheInvalidation",
+    "test": "jest",
```

Comment:
Having travis run `snyk test` was breaking the build due to travis not having the required auth

Reply:
Ah that's bad DX.

The issue is that your PR comes from your fork. Your fork does not have the secret variable used to authenticate to Snyk. Hence the build fails.

One way to handle this is to disable Snyk being run for PR builds, by testing the value of $TRAVIS_PULL_REQUEST. The upshot is that contributors can enjoy testing and green builds. The downside is that the merge build will then fail, putting load on the maintainers to silence Snyk.

Thoughts?

Reply:
> What is snyk test doing for us on travis that the snyk check isn't doing for us when a PR is opened.

You are completely right. Nothing.

> Even PR's from ourselves should be coming from forks. This hopefully reduces the 'I made a bad push' problems that are possible when pushing to the main repo. Screwing up a fork is nothing compared to breaking the main repository through a few sloppy git commands.

This is a good point, it's so convenient to push directly, but you are right, it's better to use forks.

You've convinced me, let's get rid of it and call jest instead 🙂